corehq/apps/api/resources/auth.py

```diff
@@ -97,7 +97,10 @@ def dummy(request, domain, **kwargs):
             return response
 
     def get_identifier(self, request):
```

Comment:
Just for my knowledge, is this identifier specific to the throttling? Change looks like it should work, but wasn't sure if I should be wary about anything making this identifier function differently in different contexts.

Reply:
Yes, this identifier is only used by tastypie when throttling. The only references in the tastypie codebase are here:

https://github.com/django-tastypie/django-tastypie/blob/2700cdc31bfab886149727b2f8a737309e6b27ec/tastypie/resources.py#L592

https://github.com/django-tastypie/django-tastypie/blob/2700cdc31bfab886149727b2f8a737309e6b27ec/tastypie/resources.py#L617

When I wrote this initially, I had done a deep dive to make sure the identifier wasn't used anywhere else in tastypie that would have caused us issues... then forgot to do the same in our own codebase :facepalm: